<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleSliceViewer</class>
 <widget class="QMainWindow" name="SimpleSliceViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Simple Slice Viewer</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIJ"/>
   <addaction name="actionJK"/>
   <addaction name="actionKI"/>
   <addaction name="actionPreviousSlice"/>
   <addaction name="actionNextSlice"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomIn"/>
  </widget>
  <action name="actionIJ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>IJ</string>
   </property>
  </action>
  <action name="actionJK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>JK</string>
   </property>
  </action>
  <action name="actionKI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>KI</string>
   </property>
  </action>
  <action name="actionPreviousSlice">
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="toolTip">
    <string>Go to previous slice</string>
   </property>
  </action>
  <action name="actionNextSlice">
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="toolTip">
    <string>Go to next slice</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>-</string>
   </property>
   <property name="toolTip">
    <string>zoom out</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>+</string>
   </property>
   <property name="toolTip">
    <string>zoom in</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="simpleSliceViewer.qrc"/>
 </resources>
 <connections/>
</ui>
